bears/general/ShebangBear.py

```diff
@@ -0,0 +1,33 @@
+import os
+from coalib.bears.LocalBear import LocalBear
+from coalib.results.Diff import Diff
+from coalib.results.Result import Result
+
+
+class ShebangBear(LocalBear):
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_FIX = {'Syntax'}
+
+    def run(self, filename, file):
+        """
+        Ensure that the file uses the generic Shebang operator.
+        """
+        if len(file) == 0 or '#!/usr/bin/env' in file[0]:
```

Comment:
I think this won't work if a program or an argument is defined into `env`.
Here are a couple of examples on how `env(1)` is used in a script:
```
#!/usr/bin/env bash
```
```
#!/usr/bin/env python3
```
```
#!/usr/bin/env ruby2.4 -w
```
```
#!/usr/bin/env guile --some-arguments ---and-some-more-arguments -a -fc
```